src/components/ChatLog.tsx

```diff
@@ -17,7 +17,7 @@ export default function ChatLog({ messages }: { messages: Accessor<Message[]> })
   });
 
   return (
-    <div class="flex h-full flex-col gap-4 overflow-y-scroll" ref={elementReference}>
+    <div class="flex h-full flex-col gap-4 overflow-y-auto" ref={elementReference}>
```

Comment:
I prefer `overflow-y-scroll` as it's more explicit. Unless there is something about it that gets in the way, I'd prefer that over `overflow-y-auto`.